rispy/__init__.py

```diff
@@ -1,7 +1,6 @@
 """Read and write RIS and WOK files"""
 
-__version__ = "0.6.0"
-
+from .version import __version__  # noqa
```

Comment:
getting a merge conflict; to resolve I made these changes: https://github.com/MrTango/rispy/blob/pr/34/rispy/__init__.py
<img width="631" alt="Screen Shot 2021-05-24 at 3 59 03 PM" src="https://user-images.githubusercontent.com/999952/119401052-f8d26e00-bca8-11eb-89f0-ca0f42c9c642.png">

Reply:
Done.